paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultManageScreenInteractorTest.kt

```diff
@@ -245,6 +245,38 @@ class DefaultManageScreenInteractorTest {
         }
     }
 
+    @Test
+    fun `displayableSavedPaymentMethods updated correctly when defaultPaymentMethodId updated`() {
+        val initialPaymentMethods = PaymentMethodFixtures.createCards(3)
+        runScenario(initialPaymentMethods, currentSelection = null, handleBackPressed = {}) {
```

Comment:
does `handleBackPressed` get called in this test? Wondering why you needed to add it to this scenario -- if it does get called, that seems pretty surprising

Reply:
I copy pasted this line from another test and didn't check that it wasn't a required param for runScenario